### PR TITLE
homework lesson9

### PR DESCRIPTION
**Pull request by Polina-Bril**
Homework annotations+generics

### CODE REVIEW

File: FreeIT/src/lesson9/annotations/Run.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+public class Run {
+
+	public static void main(String[] args) throws SecurityException, ReflectiveOperationException {
+
+		inspection(MyService.class);
+	}
+
+	public static void inspection(Class<?> cls) throws ReflectiveOperationException, SecurityException {
+		if (cls.isAnnotationPresent(Version.class)) {
+			MyService myS = new MyService("Changing keyboard", 70);
```
Comment (TheFallen058) [inline FreeIT/src/lesson9/annotations/Run.java]: I'm crying when I see such variable naming( please give variables more suitable names

File: FreeIT/src/lesson9/task28/ArrayBox.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+	public ArrayBox() {
+		super();
+	}
+
+	public ArrayBox(T[] array) {
+		super();
+		this.array = array;
+	}
+
+	public T get(int index) {
```
Comment (TheFallen058) [inline FreeIT/src/lesson9/task28/ArrayBox.java:18]: Please try to implement logic to exclude ArrayIndexOutOfBoundsException
options:
- check array length
- add try - catch

**Review by TheFallen058 — APPROVED** [review]
Please pay attention to my comments

File: FreeIT/src/lesson9/task28/ArrayBox.java (hunk 2 above)
Comment (Polina-Bril) [reply FreeIT/src/lesson9/task28/ArrayBox.java:18]: Correction done

File: FreeIT/src/lesson9/annotations/Run.java (hunk 1 above)
Comment (Polina-Bril) [reply FreeIT/src/lesson9/annotations/Run.java]: Sorry))) Corrected!

**Review by TheFallen058 — APPROVED** [review]
Amazing!

**Pull request merged**